src/fides/api/schemas/connection_configuration/connection_secrets_rds_mysql.py

```diff
@@ -0,0 +1,35 @@
+from typing import ClassVar, List
+
+from pydantic import Field
+
+from fides.api.schemas.base_class import NoValidationSchema
+from fides.api.schemas.connection_configuration.connection_secrets_base_aws import (
+    BaseAWSSchema,
+)
+
+
+class RDSMySQLSchema(BaseAWSSchema):
+    """
+    Schema to validate the secrets needed to connect to a RDS MySQL Database
+    """
+
+    username: str = Field(
+        default="fides_explorer",
```

Comment:
Looking at connection secrets for other integrations, we don't set defaults for `username` fields... in what way is this connection different?

Reply:
so I'm still a bit confused (perhaps just from the lack of context I have on the feature 😅 ) -- if the username is not named fides_explorer then this won't work? if so why is it configurable?